src/main/java/com/robosh/ejournal/data/dto/student/StudentSaveDto.java

```diff
@@ -0,0 +1,24 @@
+package com.robosh.ejournal.data.dto.student;
+
+import com.robosh.ejournal.data.dto.UserDto;
+import com.robosh.ejournal.data.entity.Parent;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class StudentSaveDto extends UserDto {
+
```

Comment:
It's better to name UpdateStudentDto, as it will use for update and save

Reply:
maybe it is bette to name SaveStudentDto. As example in CrudRepository there is no method update, only save which is used both for save and update

Reply:
Maybe you are right, so change please name of UpdateAdminDto

Reply:
ok!